requirements-base.txt

```
@@ -45,7 +45,7 @@ python-rapidjson>=1.4
 psutil>=5.9.2
 psycopg2-binary>=2.9.9
 PyJWT>=2.4.0
-pydantic>=1.10.17,<2
+pydantic>=2.5.0
```

```suggestion
pydantic>=2.5.0,<3
```

Could we pin to the 2.x range so that we don't accidentally bump a major?

---

there's no risk to uncapping here -- in fact it's encouraged

this file isn't installed from -- it's used to generate the pinned file